lib/vsc/mympirun/pmi/sched.py

```diff
@@ -53,8 +53,8 @@ def __init__(self, nodes=None, cores=None, ranks=None, mem=None, gpus=None):
 
     def __str__(self):
         """Human readable"""
-        return "{} nodes; with per node {} cores, {} ranks, {} mem, {} gpus".format(
-            self.nodes, self.cores, self.ranks, self.mem, self.gpus)
+        return (f"{self.nodes} nodes; with per node {self.cores} cores, "
+            "{self.ranks} ranks, {self.mem} mem, {self.gpus} gpus")
```

Comment:
you will need and `f` prefixed on this line as well, every line is handled separately

Reply:
Oh right, good catch, thanks, will fix

Reply:
fixed in force push of b400be0